Apps/Sandcastle/gallery/Spheres and Ellipsoids.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
-    <meta name="description" content="Draw spheres and ellipsoids.">
+    <meta name="description" content="Draw spheres, ellipsoids, and sensor volumes.">
```

Comment:
Very cool example.

Please update the screenshot in the same directory.

Also, let's not call these "sensor volumes" since that means something quiet different to the AGI community using Cesium.  Perhaps partial ellipsoids or ellipsoid slices?

Reply:
Also, if you want to go wild in this example, it could be cool to have an example or two that animates!

Reply:
@pjcozzi What if I change it to "sensor coverage"? A use case for this ability is to draw the coverage area of a radar, so I wanted to mention that somehow.

Reply:
Please keep this as generic capability - the Cesium team would not endorse this as sensors - as we do a lot of work in this area with fully featured sensor visibility models and can not confuse our user community.